db/schema.rb

```diff
@@ -179,8 +179,8 @@
     t.bigint "user_id", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["standup_meeting_group_id", "user_id"], name: "index_smg_users_on_smg_id_and_user_id"
-    t.index ["user_id", "standup_meeting_group_id"], name: "index_smg_users_on_user_id_and_smg_id"
+    t.index ["standup_meeting_group_id", "user_id"], name: "index_smg_users_on_smg_id_and_user_id", unique: true
+    t.index ["user_id", "standup_meeting_group_id"], name: "index_smg_users_on_user_id_and_smg_id", unique: true
```

Comment:
So when you remove a migration the steps need to go:
1. `rake db:rollback` (make sure it is before the migration was run
2. rerun `rake db:migrate`

This will ensure it is removed properly (because you never want to manually touch this file)

Reply:
Ok, I ran the rollback successfully.
Schema is back to the way it was.

Reply:
@wwrk22 I think, as long as you're addressing it in the next PR, it's fine to disable.
To disable, I'd search for the error message first to figure out the specific rubocop to disable, then add a rubocop ignore comment before for that rule and re-enable it after the offending block of code.

Reply:
Will do. Thanks for the tip, @afogel !